cocos/core/data/object.ts

```diff
@@ -452,9 +452,12 @@ if (EDITOR) {
  * issue: https://github.com/cocos/cocos-engine/issues/14643
  */
 (prototype as any)._deserialize = null;
-
-CCClass.fastDefine('cc.Object', CCObject, { _name: '', _objFlags: 0, [editorExtrasTag]: {} });
-CCClass.Attr.setClassAttr(CCObject, editorExtrasTag, 'editorOnly', true);
+if (EDITOR_NOT_IN_PREVIEW) {
```

Comment:
It is possible to pause editing while previewing, so all data should not be previewed, just use the EDITOR

Reply:
OK, I was using `EDITOR` before, I think the performance of gameview might be affected too.

Reply:
> so all data should not be previewed

Not so serious, editorExtras is currently used mainly in the animation module editor interaction